.semaphore/heroku.yml

```diff
@@ -0,0 +1,31 @@
+version: v1.0
+name: Deploy to Heroku
+agent:
+  machine:
+    type: e1-standard-2
+    os_image: ubuntu1804
+blocks:
+  - name: Deploy
+    task:
+      secrets:
+        - name: preinternship
+      env_vars:
+        - name: HEROKU_REMOTE
+          value: 'https://git.heroku.com/fierce-plains-93658.git'
+        - name: HEROKU_APP_NAME
```

Comment:
I'm not sure that HEROKU_APP_NAME &  HEROKU_API_KEY are requred keys. Could you check, please?

Reply:
HEROKU_API_KEY need for complete rails db:migrate, HEROKU_APP_NAME deleted